syndeo-template/src/main/java/com/google/cloud/syndeo/SyndeoTemplate.java

```diff
@@ -114,6 +114,44 @@ public interface Options extends PipelineOptions, DataflowPipelineOptions {
     SUPPORTED_URNS.putAll(SUPPORTED_SCALAR_TRANSFORM_URNS);
   }
 
+  public static final Map<String, String> URNS_TO_DISPLAY_NAMES =
```

Comment:
Since we've already chosen to set metadata through annotations, I would prefer to do that, and it will make expanding that metadata a bit easier going forward. We can skip setting this for now and work on adding transform-level metadata in separate PRs.

Reply:
Removed the map. Are you saying this because an annotation for the Transform name doesn't exist yet in the Beam annotations folder [here](https://github.com/apache/beam/tree/master/sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/annotations)?